gmt/tests/test_clib.py

```diff
@@ -89,18 +89,18 @@ def test_errors_sent_to_log_file():
         mode = lib.get_constant('GMT_MODULE_CMD')
         with lib.log_to_file() as logfile:
             assert os.path.exists(logfile)
-            data_file = 'not-a-valid-data-file.bla'
-            # Make a bogus module call that will fail
-            status = lib._c_call_module(lib.current_session,
-                                        'info'.encode(),
-                                        mode,
-                                        data_file.encode())
-            assert status != 0
+            with GMTTempFile() as data_file:
+                # Make a bogus module call that will fail
+                status = lib._c_call_module(lib.current_session,
+                                            'info'.encode(),
+                                            mode,
+                                            data_file.name.encode())
+                assert status != 0
```

Comment:
@seisman with my changes in #123, this version won't work because the temp file exists when it's created.

Reply:
I just changed it back to its older version.